wagtail/users/views/users.py

```diff
@@ -412,13 +418,13 @@ class UserViewSet(ModelViewSet):
     add_view_class = Create
     edit_view_class = Edit
     delete_view_class = Delete
+    history_view_class = History
```

Comment:
Ideally these views should be properly named with the `View` suffix e.g. `HistoryView`. I decided to leave this out for now, to be worked on when we add support for customising the `UserViewSet`.